src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java

```diff
@@ -277,9 +279,11 @@ private void apply(RoutingContext routingContext) {
     });
   }
 
-  private CompletableFuture<Result<JsonObject>> buildJsonResult(String policyId){
-    return CompletableFuture.completedFuture(succeeded(new JsonObject().put(getPolicyIdKey(),
-      policyId)));
+  private CompletableFuture<Result<JsonObject>> buildJsonResult(Pair<String, List<String>> policyIdAndConditionsPair){
```

Comment:
Please could this line be split

```suggestion
  private CompletableFuture<Result<JsonObject>> buildJsonResult(Pair<String, List<String>> policyIdAndConditionsPair) {
```

Reply:
done